modules/nanopolish/1.0/nanopolish.smk

```diff
@@ -102,20 +102,17 @@ rule _install_nanopolish:
     params:
         nanopolish = CFG["inputs"]["nanopolish_exec"]
     output: 
-        complete = CFG["inputs"]["nanopolish_exec"] + "/nanopolish_dependencies_installed.success"
+        complete = touch(CFG["inputs"]["nanopolish_exec"] + "/nanopolish_dependencies_installed.success")
     shell:
-        '''
-
-        if [ ! -f {params.nanopolish}/nanopolish ]; then
-            mkdir -p {params.nanopolish};
-            git clone git@github.com:jts/nanopolish.git {params.nanopolish};
-            cd {params.nanopolish};
-            make;
-            cd -;
-        fi
+        op.as_one_line('''
+        mkdir -p {params.nanopolish};
+        git clone git@github.com:jts/nanopolish.git {params.nanopolish};
+        cd {params.nanopolish};
+        make;
+        cd -;
         
-        touch  {output.complete};
-        '''
+        {output.complete};
```

Comment:
Won't this error out? I think it'll try to run `nanopolish_dependencies_installed.success` as a command, right? I think you can just delete this line altogether and end at `make`.

Reply:
Good catch! Removed {output.complete} line to fix this issue!